packages/components/src/DataTable/Header.tsx

```diff
@@ -46,14 +46,22 @@ export function Header<T extends object>({
               >
                 {header.isPlaceholder ? null : (
                   <div>
-                    {flexRender(
-                      header.column.columnDef.header,
-                      header.getContext(),
-                    )}
+                    <>
+                      {flexRender(
+                        header.column.columnDef.header,
+                        header.getContext(),
+                      )}
+                      {header.column.getCanSort() ||
```

Comment:
Had to use a turnery for this logic to work.. not sure why I couldn't use a conditional `&&` - any insight is appreciated